java/dbschema/nbproject/org-netbeans-modules-dbschema.sig

```diff
@@ -1635,7 +1635,7 @@ hfds bundle,con,dmd,driver,password,schema,url,username
 
 CLSS public org.netbeans.modules.dbschema.jdbcimpl.DBElementsCollection
 cons public init()
-fld protected static java.util.HashSet instances
+fld protected static java.util.Set<java.lang.String> instances
```

Comment:
Wow, this looks like an incompatible change, that sigtest forgot to report!

Reply:
I have [written a test](https://github.com/jtulach/netbeans-apitest/commit/eb8c08e0f7e35382e2b3ce4aa2003611678da37b) and the change seems to be properly discovered. Not sure what was going on?